static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
yeah, i think it's a bit odd because `outer` is a bad name for that thing, and i'm treating `element--inline` as the block. it's the same as the `.person--female__hand` example here, http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ but you don't see it much.
i'll rename it from `outer` so it's less confusing. thanks!